pkg/sosreport/sosreport.jsx

```diff
@@ -302,11 +301,11 @@ const SOSDialog = () => {
         <br />
         <Form isHorizontal>
             <FormGroup label={_("Report label")}>
-                <TextInput id="sos-dialog-ti-1" value={label} onChange={setLabel} />
+                <TextInput id="sos-dialog-ti-1" value={label} onChange={(_event, value) => setLabel(value)} />
             </FormGroup>
             <FormGroup label={_("Encryption passphrase")}>
                 <InputGroup>
-                    <TextInput type={showPassphrase ? "text" : "password"} value={passphrase} onChange={setPassphrase}
+                    <TextInput type={showPassphrase ? "text" : "password"} value={passphrase} onChange={(_event, value) => setPassphrase(value)}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18806-20230519-121108-54b4f0d5-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)